akka-stream-testkit/src/main/scala/akka/stream/testkit/StreamTestKit.scala

```diff
@@ -625,6 +625,17 @@ object TestSubscriber {
       self
     }
 
+    /**
+     * Fluent DSL
+     *
+     * Assert that no message is received for the specified time.
+     * Waits for the default period configured as `akka.test.expect-no-message-default`.
+     * That timeout is scaled using the configuration entry "akka.test.timefactor".
+     */
+    def expectNoMessage(): Self = {
+      expectNoMessage(probe.testKitSettings.ExpectNoMessageDefaultTimeout)
```

Comment:
shouldn't this delegate to `probe.expectNoMessage()` instead?

Reply:
Yea, that would follow the pattern. Will update.